torch_tvm/custom_tvm_ops/topi/quantize.cc

```diff
@@ -0,0 +1,89 @@
+#pragma once
+
+#include <string>
+#include <limits>
+
+#include <tvm/attrs.h>
+#include <tvm/relay/expr.h>
+
+#include "topi/reduction.h"
+#include "topi/tags.h"
+
+#include "quantize.h"
+
+namespace topi {
+using namespace tvm;
+
+Array<Tensor> data_int8_quantize(
+    const Tensor& data,
+    const Tensor& zero_point,
+    const Tensor& scale,
+    bool is_signed,
+    int precision) {
```

Comment:
[minor] If this is int8, precision is known already. Should we rename the function appropriately? Or just remove precision?

Reply:
We should allow it for fp16 in the future, for example. The name of the function needs to be generalized.